### PR TITLE
refine log

### LINKED ISSUE #8592: TiFlash should avoid log too many info logs if a query finishes quickly and normally
## Bug Report

Please answer these questions before submitting your issue. Thanks!

### PR DESCRIPTION
**Pull request by windtalker**
### What problem does this PR solve?

Issue Number: ref #6233, close #8592

Problem Summary:
Refine log, avoid to print too many info log if a mpp task finishes successfully.
### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

```release-note
Avoid printing too many info logs for successfully executed short query
```

### CODE REVIEW

File: dbms/src/Core/OperatorSpillContext.cpp [hunk 1]
```diff
@@ -47,7 +47,8 @@ void OperatorSpillContext::markSpilled()
 
 void OperatorSpillContext::finishSpillableStage()
 {
-    LOG_INFO(log, "Operator finish spill stage");
+    if (isSpillEnabled())
```
Comment (SeaRise) [inline dbms/src/Core/OperatorSpillContext.cpp:50]: how about
```suggestion
    if (is_spilled)
```

Comment (SeaRise) [reply dbms/src/Core/OperatorSpillContext.cpp:50]: just like `OperatorSpillContext::markSpilled()`

File: dbms/src/Flash/Coprocessor/DAGStorageInterpreter.cpp [hunk 2]
```diff
@@ -1467,11 +1467,13 @@ std::unordered_map<TableID, DAGStorageInterpreter::StorageWithStructureLock> DAG
             sync_schema(table_id);
         }
 
+        LOG_INFO(log, "syncing schemas done.");
```
Comment (SeaRise) [inline dbms/src/Flash/Coprocessor/DAGStorageInterpreter.cpp:1470]: add total time?

**Comment by SeaRise** [comment]
maybe the log with `finish with request unit: cpu={} read={}` can be changed to debug, Because it has been recorded in the tracing log.
And now tiflash-cn push ru to pd directly.

**Comment by SeaRise** [comment]
And some with resource control, such as `get unique_client_id succeed: {}`, etc.

**Comment by SeaRise** [comment]
Maybe the log like
```
    LOG_INFO(
        log,
        "Handling coprocessor request, is_remote_read: {}, start ts: {}, region info: {}, resource_group: {}, conn_id: "
        "{}, conn_alias: {}",
        is_remote_read,
        request->start_ts(),
        region_info,
        request->context().resource_control_context().resource_group_name(),
        request->connection_id(),
        request->connection_alias());
```
can be changed to debug if not remote read.

**Comment by windtalker** [comment]
> Maybe the log like
> 
> ```
>     LOG_INFO(
>         log,
>         "Handling coprocessor request, is_remote_read: {}, start ts: {}, region info: {}, resource_group: {}, conn_id: "
>         "{}, conn_alias: {}",
>         is_remote_read,
>         request->start_ts(),
>         region_info,
>         request->context().resource_control_context().resource_group_name(),
>         request->connection_id(),
>         request->connection_alias());
> ```
> 
> can be changed to debug if not remote read.

I change it to info because it is the only log to record conn_id and conn_alias.

File: dbms/src/Flash/Coprocessor/DAGStorageInterpreter.cpp (hunk 2 above)
Comment (windtalker) [reply dbms/src/Flash/Coprocessor/DAGStorageInterpreter.cpp]: Done

**Comment by windtalker** [comment]
> And some with resource control, such as `LOG_INFO(log, "cannot consume ru for {}, maybe it has been deleted", name);`, etc.

OK, I will confirm with @guo-shaoge

File: dbms/src/Flash/Mpp/MPPTask.cpp [hunk 3]
```diff
@@ -562,14 +562,18 @@ void MPPTask::runImpl()
...
+        LOG_IMPL(
             log,
+            log_level,
             "mpp task finish execute, is_success: {}, status: {}",
             result.is_success,
             magic_enum::enum_name(status.load()));
         auto cpu_ru = query_executor_holder->collectRequestUnit();
         auto read_ru = dag_context->getReadRU();
-        LOG_INFO(log, "mpp finish with request unit: cpu={} read={}", cpu_ru, read_ru);
+        LOG_DEBUG(log, "mpp finish with request unit: cpu={} read={}", cpu_ru, read_ru);
```
Comment (SeaRise) [inline dbms/src/Flash/Mpp/MPPTask.cpp:576]: also for cop/batchCop/streamingCop

**Review by SeaRise — APPROVED** [review]
Others LGTM

**Review by guo-shaoge — APPROVED** [review]

**Pull request merged**